.github/workflows/lint.yaml

```diff
@@ -15,6 +15,8 @@ jobs:
       - uses: actions/checkout@v4
       - name: Lint with shellcheck
         uses: ludeeus/action-shellcheck@2.0.0
+        env:
+          SHELLCHECK_OPTS: -e SC2046 -e SC2086
```

Comment:
@epiccurious What is this?

Reply:
Excluding shellcheck rules related to double-quoting variables and $() command substitution. These are two of the more frustrating ShellCheck rules. Let me know if you'd like to re-add either of them.

Here's the documentation for those two rules:
https://www.shellcheck.net/wiki/SC2046
https://www.shellcheck.net/wiki/SC2086

Reply:
Update - added two more exclude rules to this line, SC2012 and SC2196.
https://www.shellcheck.net/wiki/SC2012
https://www.shellcheck.net/wiki/SC2164